### ISSUE TITLE
Column type of aggreated column in sql view is unkown

### ISSUE THREAD

**Issue by walwe** [metabase/metabase#12547]
**Describe the bug**
Metabase fails to parse column of a view that is an aggregate such as `sum(value) as totalValue`.
Tested on SQLite.

**Logs**
```
2020-05-15T22:36:37+02:00 WARN metabase.driver.sql-jdbc.sync Don't know how to map column type '' to a Field base_type, falling back to :type/*.
2020-05-15T22:42:33+02:00 ERROR metabase.sync.util Error syncing Fields for Table 'Table 58 'testView''
```

**To Reproduce**
1. Create sample table and view in SQLite Database
```
CREATE TABLE test (
    id INTEGER not null
		primary key
		unique,
    symbol VARCHAR,
    dt DATETIME NOT NULL,
    value FLOAT NOT NULL
);

INSERT INTO test (symbol, dt, value) VALUES ("T", "2020-01-01", 0);
INSERT INTO test (symbol, dt, value) VALUES ("T", "2020-01-02", 2);
INSERT INTO test (symbol, dt, value) VALUES ("T", "2020-01-03", 4);
INSERT INTO test (symbol, dt, value) VALUES ("S", "2019-01-01", 10);

CREATE VIEW testView AS
SELECT
    symbol,
    sum(value) as totalValue
FROM test
GROUP BY symbol
ORDER by dt;
```
2. Add database to Metabase

**Expected behavior**
Column with name and type to be listed as part of view

**Information about your Metabase Installation:**
- Metabase version: v0.35.3 (latest Docker: metabase/metabase)
- Database: SQLite

**Severity**
Blocking usage of Metabase in view based scenario.

**Comment by flamber** [comment]
Hi @walwe 
It looks very similar to #12191 and #10681 - and might get fixed by #11358
Note sure if it will make a difference, but have you tried: `cast(sum(value) as float) as totalValue`

**Comment by walwe** [comment]
Thanks for the quick response.
Using cast was the first workaround I tried but does not help.

https://github.com/metabase/metabase/pull/11358 looks stale.

**Comment by walwe** [comment]
I tested https://github.com/metabase/metabase/pull/11358 and it is fixing this issue

**Comment by sbelak** [comment]
Fixed by #11358
